account_balance_line/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2010-2014 Camptocamp - Vincent Renaville
+# Copyright 2016 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Balance on journal items',
+    'summary': 'Display balance totals in move line view',
+    'version': '12.0.1.0.0',
+    'category': 'Accounting',
+    'website': 'http://www.tecnativa.com,',
```

Comment:
@davidStudio73 Should be `https://github.com/OCA/account-financial-tools`

Reply:
Changed, thanks for the tip.